jekyll/_cci2/config-policy-management-overview.adoc

```diff
@@ -148,10 +164,9 @@ enable_rule["use_official_docker_image"]
 ```
 
 ### Using Pipeline Metadata
-{: #using-pipeline-metadata }
 
-When writing policies for circleci config, it is often desired to have policies that vary slightly in behaviour by project or branch.
-This is possible using the `data.meta` rego property access. When a policy is evaluated in the context of a triggered pipeline the following three properties will be available on `data.meta`: 
+When writing policies for circleci config, it is often desired to have policies that vary slightly in behaviour by project or branch. This is possible using the `data.meta` rego property access. When a policy is evaluated in the context of a triggered pipeline the following three properties will be available on `data.meta`: 
+
 ```
 project_id    (CircleCI Project UUID)
 branch        (string)
```

Comment:
What is `build_number`?

Reply:
It's the id of the build. For example, [here's](https://app.circleci.com/pipelines/github/circleci/circleci-docs/45445/workflows/6bf6947f-9d7b-4528-b841-aa619a8e7191/jobs/143322) a link for a build from this PR. The number at the end of the URL is the build number

Reply:
Ah right, we call this the job number

Reply:
the metadata is actually `build_number`?

Reply:
For example: https://circleci.com/docs/api/v2/index.html#operation/getJobDetails (job number) - just to check we're talking about the same thing

Reply:
Yes they are the same. job_number and build_number. I guess there are two different names for the same thing... We could add an alias `job_number` that would be the same as `build_number` if that would be helpful.

The metadata can be found on the key `meta` of the global object `data`. It looks like this:

```
data:
  meta:
     build_number: (int)
     project_id:       (string/uuid)
     branch:            (string)
```